actions/upload_prod_artefacts/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Upload production artefact (not publish)'
+description: 'Upload production artefact (not publish)'
+inputs:
+  upload_prefix:
+    description: 'upload name prefix'
```

Comment:
example of what the prefix could be?

Reply:
this is not used by anything else than CI